app/models/planet.py

```diff
@@ -0,0 +1,20 @@
+class Planet:
+    def __init__(self, id, name, description, size_dia_km):
+        self.id = id
+        self.name = name
+        self.description = description
+        self.size = size_dia_km
+
+    def to_dict(self):
+        return dict(
+            id=self.id,
+            name=self.name,
+            desc=self.description,
```

Comment:
Unless there's a strong need to, resist the temptation to shorten the name in the dictionary-ified version.